src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule73wherejavadocrequired/InputMissingJavadocTypeCheckNoViolations.java

```diff
@@ -0,0 +1,8 @@
+package com.google.checkstyle.test.chapter7javadoc.rule73wherejavadocrequired;
+
+/**
+ * This is a Javadoc comment.
+ */
+public class InputMissingJavadocTypeCheckNoViolations { // OK
```

Comment:
please place public enum, inteface without javadoc ... to clearly state it is not required.
I do believe it is mistake in style but we need to wait until it is resolved in style first.

Reply:
Done.

Reply:
From https://google.github.io/styleguide/javaguide.html#s1.1-terminology:
"The term class is used inclusively to mean an "ordinary" class, enum class, interface or annotation type (@interface)."

Therefore, google style dictates that we should use the default tokens for this check.  I am reworking this PR to reflect this.

Reply:
So we need to add corresponding tokens in config